AdminWebsite/AdminWebsite.UnitTests/Helper/AdministratorRoleClaimsHelperTest.cs

```diff
@@ -0,0 +1,91 @@
+﻿using AdminWebsite.Helper;
+using AdminWebsite.Services.Models;
+using FluentAssertions;
+using NUnit.Framework;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+
+namespace AdminWebsite.UnitTests.Helper
+{
+    public class AdministratorRoleClaimsHelperTest
+    {
+        [Test]
+        public void GetAdministratorClaims_Returns_Claims()
```

Comment:
I know you asked about this before and I don't think I gave a proper answer, since we're following the `should_something_when/given_something` pattern can we keep that format on all the tests?

Reply:
done